Automattic/pocket-casts-android
# Display End of Year stories skeleton UI

## Description

This PR maps EoY stats to stories and pushes them to a basic UI.

Figma with all stories listed: lH66LwxxgG8btQ8NrM0ldx-fi-3070_28391

## Testing Instructions

1. Sign in with a paid account that listened to at least 5 minutes of episodes this year.
2. Sync your profile data.
3. Close the app.
4. Reopen the app.
5. Open Playback 2024 either by using the bottom sheet or the banner in the profile.
6. You might see a loading progress bar and it might take some time to sync your data.
7. Once data is synced tapping on the right edge side of a story should move you forward and tapping on a left edge of a story should move you backwards.
8. Verify that you can see all stories except for the `PlusInterstitial`.
9. Sign out.
10. Sign in with a non-paid account.
11. Verify that you can see the `PlusInterstitial` story.

## Checklist
